Hyperbolic_surface_triangulation_2/test/Hyperbolic_surface_triangulation_2/hs_test_complex.cpp

```diff
@@ -1,16 +1,15 @@
-#include <CGAL/Complex_without_sqrt.h>
+#include <CGAL/Complex_number.h>
 #include <CGAL/Exact_rational.h>
-#include <CGAL/Exact_integer.h>
 #include <CGAL/Interval_nt.h>
 #include <iostream>
 #include <sstream>
 #include <complex>
 
 using namespace CGAL;
 
-typedef Complex_without_sqrt<Gmpq> Complex_gmpq;
-typedef Complex_without_sqrt<Exact_integer> Complex_integer;
-typedef Complex_without_sqrt<Interval_nt<>> Complex_interval;
+typedef Complex_number<Gmpq> Complex_gmpq;
```

Comment:
No need to test with `Gmpq`.  There will be test platforms where `Exact_rational`  is this type.  And there will be plarforms where `GMP`  is not even installed.